app/controllers/concerns/fraud_review_concern.rb

```diff
@@ -22,13 +22,13 @@ def handle_pending_fraud_review
   end
 
   def handle_fraud_rejection
-    return if in_person_prevent_fraud_redirection?
```

Comment:
Talking through this some more just for posterity: the idea here is that in the case of fraud _rejection_, the question of whether to redirect them to the please-call page is irrelevant, because that's really for the pending state, not the rejection state.

Reply:
@n1zyy my thinking was incorrect at the time, see [this slack thread](https://gsa-tts.slack.com/archives/C03FA4VBN76/p1710973042439869?thread_ts=1710972257.115159&cid=C03FA4VBN76). note that the changes I made are designed to handle the case where someone fails at the post office, and then is fraud rejected, which is super unlikely given that the user will not be shown the Please Call screen.